packages/@webex/plugin-meetings/test/integration/spec/journey.js

```diff
@@ -239,7 +239,7 @@ skipInNode(describe)('plugin-meetings', () => {
 
     // Enabled when config.enableUnifiedMeetings = true
     xdescribe('Conversation URL', () => {
-      describe('Successful 1:1 meeting', () => {
+      describe('Successful 1:1 meeting', async () => {
```

Comment:
Why async in describe? Where do we await for this?

Reply:
I missed this. This is not required. Removed